lib/client.js

```diff
@@ -44,6 +44,8 @@ exports.boot = function boot(options, callback) {
 
   var useRouter = (props.__meta.view === null);
 
+  var mountNode = options.mountNode || _document;
```

Comment:
awesome. but what about the server side? did you try an example on the server side?